src/presentation/http/router/note.test.ts

```diff
@@ -49,4 +52,67 @@ describe('Note API', () => {
       expect(body).toStrictEqual({ message: 'Note not found' });
     });
   });
-});
+
+  describe('GET note/:notePublicId ', () => {
+    test('Returns note by public id with 200 status', async () => {
+      const expectedStatus = 200;
+
+      const expectedNote = {
+        'id': 2,
+        'publicId': 'Pq1T9vc23Q',
+        'creatorId': 1,
+        'content': null,
+        'createdAt': '2023-10-16T13:49:19.000Z',
+        'updatedAt': '2023-10-16T13:49:19.000Z',
+      };
+
+      const response = await global.api?.fakeRequest({
+        method: 'GET',
+        url: '/note/Pq1T9vc23Q',
+      });
+
+      expect(response?.statusCode).toBe(expectedStatus);
+
+      const body = response?.body !== undefined ? JSON.parse(response?.body) : {};
+
+      expect(body).toStrictEqual(expectedNote);
```

Comment:
try using `response?.json()` instead of manually parsing body here

Reply:
That's much better, thank you!